README.md

```diff
@@ -120,6 +120,7 @@ func main() {
 * warn (`zerolog.WarnLevel`, 2)
 * info (`zerolog.InfoLevel`, 1)
 * debug (`zerolog.DebugLevel`, 0)
+* trace (`zerolog.TraceLevel`, 0)
```

Comment:
Don't really know how to handle an untyped int for this new level.

Reply:
Probably it makes sense to shift all log level + 1 and start with trace.
Question is if the actual number is used directly by applications using zerolog.
Usually you should use the constants when referring to specific levels.

So Trace = 0, Debug = 1, ... should not impact to many, but can be considered a breaking change.

Reply:
Exactly. I have started this feature by shifting all log level like you said but I thought it would have a big impact on applications that use it directly.
And yeah we could mark this as breaking change :)